### PR TITLE
Improve bbe code formatting

### PR DESCRIPTION
**Pull request by KRVPerera**
## Purpose
Improve BBE code formatting
- Remove extra spaces before and after braces
- Remove unsupported feature comments about out of body function definition on records

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/object-methods/object_methods.bal [hunk 1]
```diff
@@ -13,7 +13,7 @@ type Person object {
         self.lastName = lastName;
     }
 
-    // A method defined within the object.
+    // A method.
```
Comment (chamil321) [inline examples/object-methods/object_methods.bal]: Can we make this comment more descriptive? If not let's remove this comment as it's quite obvious.

Comment (KRVPerera) [reply examples/object-methods/object_methods.bal]: This is a BBE about object methods. Made it more descriptive.

File: examples/object-methods/object_methods.bal [hunk 2]
```diff
@@ -13,7 +13,7 @@ type Person object {
         self.lastName = lastName;
     }
 
-    // A method defined within the object.
+    // A method returning full name of `Person` object.
```
Comment (pubudu91) [inline examples/object-methods/object_methods.bal]:
```suggestion
    // A method returning the full name of a `Person` object.
```

Comment (praneesha) [inline examples/object-methods/object_methods.bal]:
```suggestion
    // A method returning the full name value of the `Person` object.
```

**Pull request merged**